testsuite/classlibrary/TestControlSpec.sc

```diff
@@ -28,22 +28,112 @@ TestControlSpec : UnitTest {
 		this.assert(z.minval == 70,"setting minval on spec should work");
 
 		this.assert( w.minval != 70,"setting minval on spec should not affect its copy");
-		
+
 		// and here is the bug itself
 		this.assert( w.map(0.0) == 50,"mapping using the copied spec should result in the original mapping behavior of the original spec.");
 	}
 
+	test_setFrom {
```

Comment:
I'm personally fine with those, though Brian may insist (and perhaps he is right) that we should have only one assert per method. I am undecided.

Reply:
In a way that would be correct, but also making tests clear and concise is more correct - I'm not adding 29 methods. :)
Adding descriptive messages to each to make failures more clear.

Reply:
@telephon - I'm open to discussing changes to the unit testing standards I proposed! Here I think it's reasonable to have more than one assert, because logically what's being tested is the postcondition of `setFrom`.